src/gfn/losses/sub_trajectory_balance.py

```diff
@@ -92,7 +86,7 @@ def cumulative_logprobs(
 
     def get_scores(
         self, trajectories: Trajectories
-    ) -> Tuple[List[ScoresTensor], List[ScoresTensor]]:
+    ) -> Tuple[List[TType[-1, float]], List[TType[-1, float]]]:  # TODO: why -1 here? Is it equivilant to 0?
```

Comment:
Another example where I think readability suffers.

Reply:
Right... I'd argue that the readability issue is more due to `Tuple[List[...], List[...]]` than `TType`.